tests/tests.js

```diff
@@ -2,7 +2,7 @@ var assert = require('assert');
 var assertEquals = assert.equal;
 var assertThrows = assert['throws'];
 
-require('../includes.js');
+require('../auto.js');
```

Comment:
```suggestion
require('../auto');
```
also since that's what's in "exports"